CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
+### Fixed
+- **Breaking change!** - Waveform tools - Added a missing 2pi factor into `detuning` parameter in `drag_gaussian_pulse_waveforms` and `drag_cosine_pulse_waveforms`. 
+  This will produce different results compared to previous versions, to get the same results, divide the `detuning` parameter by 2pi.
```

Comment:
This now needs to be changed also for the anharmonicity.
But I'm not sure about this change, why not just change the docstring to indicate that we expect it in Hz (and not in Rad)?

Reply:
Thanks, I updated.

Reply:
The docstrings mention already that the units should be in Hz

Reply:
I mean:
This is a breaking change, anyone updating qualang tools will have their script not working and will wonder why.
99.9% of the people do not look at the changelog... Did you ever look at the changelog when you updated Numpy? Scipy? qm-qua?? :) 
How important is it to have this change? Can't we update the docstring **instead** of doing this change?
Options we have:
1. Keep the breaking change as it is now, I don't really like this option.
2. Revert this breaking change completely, updating the docstring
3. We can use the changed variable name: If the user-supplied `delta` it will be in Rad/s and if the user gave `Anharmonicity` then it will be in Hz. But this means we need to change the function call a little bit and we also need to change the name of the detuning parameter...

WDYT? @TheoLaudatQM @KevinAVR 

I was wrong in my previous comment, I meant to write that we expect it in Rad/s and not in Hz.

Reply:
I like the third option.

Reply:
I don't follow why you need to change the name of the detuning parameter

Reply:
I'll elaborate more with a code example tomorrow

Reply:
I'm commenting outside of the "conversation" for easier finding later